runtime/doc/options.txt

```diff
@@ -3499,7 +3499,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	if you want to use Vim as a modeless editor.
 	These Insert mode commands will be useful:
 	- Use the cursor keys to move around.
-	- Use CTRL-O to execute one Normal mode command |i_CTRL-O|).  When
+	- Use CTRL-O to execute one Normal mode command |i_CTRL-O|.  When
 	  this is a mapping, it is executed as if 'insertmode' was off.
```

Comment:
- I intentionally left out the `c_CTRL-N` 'incsearch' stuff because it will be overridden later anyways (and we don't have that patch yet).
- We default 'langremap' to "off" so I thought it best to leave out frantic attempts to help the user overcome bad defaults.